addons/mail/static/src/js/owl/component/thread.js

```diff
@@ -176,10 +221,6 @@ Thread.props = {
         type: Number,
```

Comment:
rename to `initialMessageListScrollTop`

Reply:
I propose `messageListInitialScrollTop` to be consistent with `composerInitialAttachmentLocalIds` and `composerInitialTextInputHtmlContent`

Reply:
ok